Examples/Add-UiPathDefaultRoles.ps1

```diff
@@ -0,0 +1,56 @@
+$URL = Read-Host -Prompt 'Orchestrator URL'
```

Comment:
Authentication should be handled outside the script. This breaks many other modes of connecting (Windows) and does not allow selecting tenant and/or OU. User shoudl authentiocate the PoSh session before calling this script.

Reply:
Done.